OTKit/README.md

```diff
@@ -49,6 +50,16 @@ If you are using a token in your project, you can execute `npm link '<token-name
 
 ***
 
+### Publish
+
+A token needs to be published in order for the changes to take effect. To publish a token, navigate to the token's directory, for example `./OTKit/otkit-colors/`, then run `npm publish otkit-colors`.
+
+You need to have an NPM account and be among the "Collaborators" list on the official NPM package page, for example  `https://www.npmjs.com/package/otkit-colors`, to publish. Contact one of the collaborators if you would like to be added.
+
+Note: if you are a developer of OpenTable organization, remember to remove the OpenTable credentials from your `~/.npmrc` file to make sure you are publishing to the public instead of a private registry.
```

Comment:
I think we can automate this by adding the public registry as publishUrl in the package.json? @nickbalestra can confirm

Reply:
Totally, you could simply add a ‘.npmrc’ file inside [each](https://github.com/lerna/lerna/issues/296#issuecomment-261080345) package

Reply:
I was actually suggesting using the package.json: https://docs.npmjs.com/files/package.json#publishconfig
```js
...
"publishConfig": { "registry": "...publicregistry" },
```

Reply:
btw @karuto this is a little detail that shouldn't block the PR to be merged - we can improve later ;)

Reply:
@nickbalestra @matteofigus Thanks for the review.

Oh it would awesome if we could automate registry control at a package level.